app/routes/events/components/EventDetail/index.tsx

```diff
@@ -326,27 +348,37 @@ export default class EventDetail extends Component<Props, State> {
       event.responsibleGroup && {
         key: 'Arrangør',
         value: (
-          <span>
-            {groupLink ? (
-              <Link to={groupLink}>{event.responsibleGroup.name}</Link>
-            ) : (
-              event.responsibleGroup.name
+          <Link to={groupLink} className={styles.organizerLogo}>
+            {groupLink && (
+              <Tooltip
+                key={event.responsibleGroup.id}
+                content={event.responsibleGroup.name}
+              >
+                <CircularPicture
+                  alt={event.responsibleGroup.name}
+                  src={groupLogoDict[event.responsibleGroup.name]}
+                  size={40}
+                />
+              </Tooltip>
             )}{' '}
-            {event.responsibleGroup.contactEmail && (
-              <a href={`mailto:${event.responsibleGroup.contactEmail}`}>
-                {event.responsibleGroup.contactEmail}
-              </a>
-            )}
-          </span>
+            {event.responsibleGroup.name}
+          </Link>
         ),
       },
       event?.createdBy
         ? event.createdBy && {
             key: 'Forfatter',
             value: (
-              <Link to={`/users/${event.createdBy.username}`}>
-                {event.createdBy.fullName}
-              </Link>
+              <span className={styles.organizerLogo}>
+                <ProfilePicture
+                  user={event.createdBy}
+                  size={40}
+                  alt={event.createdBy.firstName}
```

Comment:
Why not use the full name? A lot of people in this organization has the same name :)

Reply:
I chose to remove the arranger picture. So the name will be as I was before